test/jdk/javax/sound/midi/Synthesizer/Receiver/bug6186488.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2016, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You don't need a third year, but instead the second year should be updated.

Reply:
Updated the year.